File: content/03/code/dotnet/07--step07-use-person-parameter-in-second-step/Shouty/Shouty/Person.cs
using System;
using System.Collections.Generic;

namespace Shouty
{
    public class Person
    {
        public string Name { get; }

        public Person(string name)
        {
            Name = name;
        }

        public void MoveTo(int distance)
        {
        }

        public void Shout(string message)
        {
        }

        public IList<string> GetMessagesHeard()
        {
            return new List<string> { "free bagels at Sean's" };
        }
    }
}

File: content/bdd-with-cucumber/03/code/java/07--step07-use-person-parameter-in-second-step/shouty/src/main/java/shouty/Person.java
package shouty;

import java.util.ArrayList;
import java.util.List;

public class Person {
    private Integer location;
    private String name;

    public Person(String name_) { name = name_; }

    public void moveTo(Integer location) {
        this.location = location;
    }

    public void shout(String message) {

    }

    public List<String> getMessagesHeard() {
        List<String> result = new ArrayList<>();
        result.add("free bagels at Sean's");
        return result;
    }
}
